curvefs/src/client/s3/disk_cache_manager.cpp

```diff
@@ -166,6 +175,10 @@ std::string DiskCacheManager::GetCacheWriteFullDir() {
 
 int DiskCacheManager::WriteDiskFile(const std::string fileName, const char *buf,
                                     uint64_t length, bool force) {
+    // write bps throttle
+    diskCacheThrottle_.Add(false, length);
+    // write iops throttle
+    diskCacheThrottle_.Add(false, 1);
```

Comment:
only `diskCacheThrottle_.Add(false, length);` is enough, it will add 1 token to iops bucket and #length tokens to bps bucket internally.

Reply:
fix